recipes/wt/all/conanfile.py

```diff
@@ -0,0 +1,153 @@
+        "with_firebird": [True, False],
+        "with_mysql": [True, False],
+        "with_mssql": [True, False],
+        "with_qt4": [True, False],
+        "with_test": [True, False],
+        "with_dbo": [True, False],
+        "with_opengl": [True, False],
+        "with_unwind": [True, False],
+        "no_std_locale": [True, False],
+        "no_std_wstring": [True, False],
+        "multi_threaded": [True, False],
+        "connector_http": [True, False],
+        "connector_isapi": [True, False],
+        "connector_fcgi": [True, False]
+    }
+    default_options = {'shared': False, 'fPIC': True, 'with_ssl': True, 'with_haru': False, 'with_pango': False, 'with_sqlite': True, 'with_postgres': False, 'with_firebird': False, 'with_mysql': False, 'with_mssql': False, 'with_qt4': False, 'with_test': True, 'with_dbo': True, 'with_opengl': False, 'with_unwind': False, 'no_std_locale': False, 'no_std_wstring': False, 'multi_threaded': True, 'connector_http': True, 'connector_isapi': True, 'connector_fcgi': False}
+
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+
+    _cmake = None
+
+    requires = ('zlib/1.2.11', 'boost/1.73.0')
+
+    def requirements(self):
+        if self.options.with_ssl:
+            self.requires('openssl/1.1.1g')
+        if self.options.with_sqlite:
+            self.requires('sqlite3/3.31.1')
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+            del self.options.connector_fcgi
+        else:
+            del self.options.connector_isapi
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions['CONFIGDIR'] = os.path.join(self.package_folder, 'bin')
+        self._cmake.definitions['SHARED_LIBS'] = self.options.shared
+        self._cmake.definitions['BUILD_EXAMPLES'] = False
+        self._cmake.definitions['BUILD_TESTS'] = False
+        self._cmake.definitions['ENABLE_SSL'] = self.options.with_ssl
+        self._cmake.definitions['ENABLE_HARU'] = self.options.with_haru
+        self._cmake.definitions['ENABLE_PANGO'] = self.options.with_pango
+        self._cmake.definitions['ENABLE_SQLITE'] = self.options.with_sqlite
+        self._cmake.definitions['ENABLE_POSTGRES'] = self.options.with_postgres
+        self._cmake.definitions['ENABLE_FIREBIRD'] = self.options.with_firebird
+        self._cmake.definitions['ENABLE_MYSQL'] = self.options.with_mysql
+        self._cmake.definitions['ENABLE_MSSQLSERVER'] = self.options.with_mssql
+        self._cmake.definitions['ENABLE_QT4'] = self.options.with_qt4
+        self._cmake.definitions['ENABLE_LIBWTTEST'] = self.options.with_test
```

Comment:
This seems to be a selftest:
https://github.com/emweb/wt/blob/34addc77d805916a9aa41c9a2beac15c4a49899d/test/CMakeLists.txt#L1

Reply:
it is used in selftest, but it's a set of class part of public api, which can be reused independantly of self tests: https://github.com/emweb/wt/blob/247fdd1f1e8ea407fd8a65fa8de7eee21f0d7daa/CMakeLists.txt#L43 
https://webtoolkit.eu/wt/doc/reference/html/classWt_1_1Test_1_1WTestEnvironment.html

Reply:
Indeed. Thanks!
Users should be able to test their servers.